packages/eslint-plugin-pf-codemods/lib/rules/v5/nav-warn-flyouts-now-inline.js

```diff
@@ -0,0 +1,33 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8628
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
+      (specifier) => specifier.imported.name === "Nav"
+    );
+
+    return imports.length === 0
+      ? {}
+      : {
+          JSXOpeningElement(node) {
+            const hasNav = imports
+              .map((imp) => imp.local.name)
```

Comment:
as we should only have the `Nav` import, i think we should just call the `import` `navImport` with a `find` instead of `filter` and not have to map on this